grpc/server/admin_test.go

```diff
@@ -29,6 +30,72 @@ func newTestAdminServer(t *testing.T) (*AdminServer, *db.MockQuerier) {
 	}, mockQuerier
 }
 
+func TestAdminServer_IssueAccessToken(t *testing.T) {
+	tests := []struct {
+		name              string
+		token             *commonv1.AccessToken
+		errCode           codes.Code
```

Comment:
I realized I missed this in the last PR. I think this would make more sense to be called the response code right? It doesn't always represent the response code for an error.

Reply:
Agree. Fixed!